appconfig/appconfig.go

```diff
@@ -0,0 +1,100 @@
+// Package appconfig loads repository configuration for GitHub apps. It
+// supports loading directly from a file in a repository, loading from remote
+// references, and loading an organization-level default. The config itself can
+// be in any format.
+package appconfig
```

Comment:
I think this is a reasonable package name. I agree that `config` is probably too generic.